Extensions/find_inactives.js

```diff
@@ -138,13 +136,19 @@ XKit.extensions.find_inactives = new Object({
 
 
 						if (last_updated_days >= parseInt(XKit.extensions.find_inactives.preferences.time.value)) {
-							var m_user = {};
-							m_user.username = username;
-							m_user.days = last_updated;
+							var m_user = {
+								username: username,
+								days: last_updated,
+								days_value: last_updated_days
+							};
 
 							XKit.extensions.find_inactives.retired_people_list.push(m_user);
 						}
+					});
 
+					// Sort inactive blogs -- Oldest first
+					XKit.extensions.find_inactives.retired_people_list.sort(function(blog_a, blog_b) {
+						return -(blog_a.days_value - blog_b.days_value);
```

Comment:
nit: This would be cleaner as `blog_b.days_value - blog_a.days_value` instead of the double minus